Clean up oasis managers (SOFTWARE-4279)

brianhlin opened the pull request:
@DrDaveD cleaning up OASIS managers that have not registered in COManage after giving them fair warning and a ~2 week grace period (apart from some managers that we did not contact). I followed the instructions in https://opensciencegrid.atlassian.net/browse/OPS-35 and checked the contents of the VO dirs on `oasis-login` and marked `UseOASIS: false` for VOs without managers + empty dirs.

DrDaveD commented:
I don't think we should drop the managers from any VOs that are getting actively updated.  There are 5 that are recently updated:
```
$ ls -ltr /cvmfs/oasis.opensciencegrid.org/*/update.details|tail
-rw-r--r-- 1 cvmfs cvmfs 279 Aug  8  2016 /cvmfs/oasis.opensciencegrid.org/nanohub/update.details
-rw-rw-r-- 1 cvmfs cvmfs 295 Mar 24  2017 /cvmfs/oasis.opensciencegrid.org/sbgrid/update.details
-rw-rw-r-- 1 cvmfs cvmfs 284 Feb 20  2018 /cvmfs/oasis.opensciencegrid.org/geant4/update.details
-rw-rw-r-- 1 cvmfs cvmfs 139 May  3  2018 /cvmfs/oasis.opensciencegrid.org/fermilab/update.details
-rw-rw-r-- 1 cvmfs cvmfs 278 Jun 23  2020 /cvmfs/oasis.opensciencegrid.org/osg/update.details
-rw-r-xr-x 1 cvmfs cvmfs 194 Jul 13 00:04 /cvmfs/oasis.opensciencegrid.org/gluex/update.details
-rw-r--r-- 1 cvmfs cvmfs 210 Jul 13 03:25 /cvmfs/oasis.opensciencegrid.org/accre/update.details
-rw-rw-r-- 1 cvmfs cvmfs 202 Jul 13 11:53 /cvmfs/oasis.opensciencegrid.org/ligo/update.details
-rw-rw-r-- 1 cvmfs cvmfs 166 Jul 13 13:18 /cvmfs/oasis.opensciencegrid.org/mis/update.details
-rw-r--r-- 1 cvmfs cvmfs 193 Jul 13 15:06 /cvmfs/oasis.opensciencegrid.org/jlab/update.details
```

DrDaveD commented on virtual-organizations/CMS.yaml:
```diff
@@ -34,21 +34,8 @@ ID: 3
   OASISRepoURLs:
-  - http://oasiscfs.fnal.gov:8000/cvmfs/cms-lpc.opensciencegrid.org
+    - http://oasiscfs.fnal.gov:8000/cvmfs/cms-lpc.opensciencegrid.org
   UseOASIS: true
```
The cms dir is empty, you could set UseOASIS: false

brianhlin replied:
Done

DrDaveD commented on virtual-organizations/auger.yaml:
```diff
@@ -20,13 +20,8 @@ ID: 102
   OASISRepoURLs:
-  - http://cvmfs-stratum0.gridpp.rl.ac.uk:8000/cvmfs/auger.egi.eu
+    - http://cvmfs-stratum0.gridpp.rl.ac.uk:8000/cvmfs/auger.egi.eu
   UseOASIS: true
```
I think this one can be false too, they only have one test file.

brianhlin replied:
Done

DrDaveD commented:
I did `grep -i "UseOASIS: true"|wc -l` and counted 53 VOs.   The only VO directories that have anything possibly worth keeping are:
```
accre
csiu
enmr
geant4
glow
gluex
ilc
jlab
ligo
mis
nanohub
osg
sbgrid
snoplussnolabca
```
So you must be missing a bunch of them for the cleanup.   It's likely quite a few of this shorter list which hasn't been updated in years also could be removed, but maybe that's a separate cleanup effort.

brianhlin commented:
@DrDaveD I've addressed your comments, how does this PR look now?

> So you must be missing a bunch of them for the cleanup.

Yup, I'm not surprised. I didn't actually do an audit of every VO's contents on `oasis-login` and just based this off of COManage registrations. There were also a few managers who are no longer in the contact DB that I still need to contact separately so there will likely be a second round of manager-related cleanup.

How did you determine whether a VO had anything worth keeping on `oasis-login`? That definitely seems like another cleanup worth doing.

DrDaveD commented:
I just looked in the directories on /cvmfs/oasis.opensciencegrid.org and did not include the ones that were empty, did not match a VO name, or only had one very old test file.

DrDaveD reviewed (approved):
These changes are ok.  Getting rid of the remaining `UseOASIS: true` will be done later.
